source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -158,54 +158,34 @@ Configure and run CRI-O + kata-runtime
 
        sudo kubeadm init --cri-socket=/run/crio/crio.sock
 
-
 Install pod network add-on
 **************************
 
 You must choose and install a `pod network add-on`_ to allow your pods to
 communicate. Check whether or not your add-on requires special flags when you
 initialize the master control plane.
 
-**Notes about flannel add-on**
-
-If you choose the `flannel` add-on, then you must add the following to the
-`kubeadm init` command:
-
-..  code-block:: bash
-
-    --pod-network-cidr 10.244.0.0/16
-
-If you are using CRI-O and `flannel` and you want to use Kata Containers, edit the :file:`/etc/crio/crio.conf` file to add:
-
-..  code-block:: bash
-
-    [crio.runtime]
-    manage_network_ns_lifecycle = true
-
-Create a symlink for the network overlays:
-
-.. code-block:: bash
-
-   sudo ln -s /usr/libexec/cni /opt/cni/bin
+The CRI-O default plugin_dir is :file:`/opt/cni/bin`. This must be a
+writable directory because third-party networking add-ons will install
+themselves there.
 
 .. note::
 
-   |CL| installs CNI plugins that are part of the `cloud-native-basic`
-   bundle to :file:`/usr/libexec/cni`. The directory is required because `
-   swupd verify` may use it to repair a system to a known good state.
-
-**Notes about Weave Net add-on**
+   CNI plugins provided by |CL| are installed as part of *cloud-native-basic*
+   in :file:`/usr/libexec/cni/` and are currently *not* found by CRI-O by
+   default. These separate directories are required because `swupd` controls
+   the content of :file:`/usr` and leaves :file:`/opt` unchanged.
 
-If you choose the `Weave Net` add-on, you must make the following
-changes because it installs itself in the :file:`/opt/cni/bin` directory.
+When using third-party network add-ons that rely on those plugins, such as
+Weave or Flannel do, make them available by creating symlinks:
 
-For using CRI-O and ``Weave Net``, complete the following step.
+.. code-block:: bash
 
-Add the `loopback` CNI plugin to the plugin path with the command:
+   sudo mkdir -p /opt/cni/bin
 
 .. code-block:: bash
 
-   sudo ln -s /usr/libexec/cni/loopback /opt/bin/cni/loopback
+   for i in /usr/libexec/cni/*; do sudo ln -sf $i /opt/bin/cni/; done
```

Comment:
should be ```/opt/cni/bin```! :)

Reply:
See latest commit please. Please verify. [Sorry but I don't have time to validate this on a test box.]